src/server/test_utils.cc

```diff
@@ -209,13 +209,16 @@ RespExpr BaseFamilyTest::Run(absl::Span<std::string> span) {
 RespExpr BaseFamilyTest::Run(std::string_view id, ArgSlice slice) {
   TestConnWrapper* conn_wrapper = AddFindConn(Protocol::REDIS, id);
 
+  conn_wrapper->conn()->SetAdmin(admin_);
+
   CmdArgVec args = conn_wrapper->Args(slice);
 
   auto* context = conn_wrapper->cmd_cntx();
 
   DCHECK(context->transaction == nullptr) << id;
 
   service_->DispatchCommand(CmdArgList{args}, context);
+  conn_wrapper->conn()->SetAdmin(false);
```

Comment:
This is this needed?
Please document as it's unintuitive

Reply:
I will document this in code. The reason for it is that we use RunAdmin to use admin connection, but connections are saved in a map where we take connections from. So if we want the next call to Run() not use admin connection we must disable it